config/remote/remote_config.go

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (C) 2017 The "MysteriumNetwork/node" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package remote_config
+
+import (
+	"math/big"
+	"strings"
+
+	"github.com/urfave/cli/v2"
+
+	"github.com/spf13/cast"
+
+	"github.com/mysteriumnetwork/node/tequilapi/client"
+
+	"github.com/mysteriumnetwork/node/config"
+	"github.com/rs/zerolog/log"
+)
+
+var Config = newRemoteConfig()
```

Comment:
I am not a big fan of these sort of global variables. Whats the issue with exposing `NewRemoteConfig` and allowing the user to manipulate it how ever he wants to?

It's just much easier to read/edit when things are not hidden in packages. It's also not very static, someone calling `Get` while you're calling `RefreshRemoteConfig` might get nasty.

Reply:
It is called a "singleton pattern" and I am not a fan of them too.
But since I wan't to get a stable MVP the idea was to copy past in a similar fashion and then test the solution until we are satisfied it is stable. However maybe this is not a big change.